spec/coder_spec.lua

```diff
@@ -1497,6 +1545,15 @@ describe("Pallene coder /", function()
         end)
     end)
 
+    describe("ipairs builtin", function()
+        it("work", function()
+            run_test([[
+                local t = {10, 20, 30}
+                assert(ipairs(t))
```

Comment:
This test is just calling the Lua ipairs function, without calling any pallene functions from the `test` module.

It would be nice to have a test cast describing what happens if someone tried to call ipairs in Pallene outside of a for-in loop.